src/extension/mapml/src/main/java/org/geoserver/mapml/MapMLConstants.java

```diff
@@ -31,8 +31,13 @@ public final class MapMLConstants {
     /** format name */
     public static final String FORMAT_NAME = "MAPML";
 
-    /** MapML format options */
-    public static final String MAPML_FEATURE_FORMAT_OPTIONS = "mapmlfeatures:true";
+    /** MapML format option enabling features */
+    public static final String MAPML_FEATURE_FO = "mapmlfeatures";
+
+    /** MapML format option enabling attribute skipping */
+    public static final String MAPML_SKIP_ATTRIBUTES_FO = "mapmlskipattributes";
+
+    public static final String MAPML_SKIP_STYLES_FO = "mapmlskiphstyles";
```

Comment:
`mapmlskiphstyles` possibly a typo? -> `mapmlskipstyles`

Reply:
Not a typo, skipstyles would imply, in my mind, just wireframe, without any style. What we don't want is style definitions in the "head", while keeping the styles references in the features. 
"skipheadstyles" seemed too long though but quite a bit clearer. 
Open to suggestions.

Reply:
I have been opting for longer but obvious variable names recently as I'm getting older and seem to always forget what I was up to when I originally wrote some piece of code in the past

Reply:
Switched to the longer name